spec/support/auth_support.rb

```diff
@@ -14,9 +14,10 @@ def compmgr_stub_auth
     stub_auth
   end
 
-  def certmgr_stub_auth
+  def certmgr_stub_auth(team = nil)
     @request.env["devise.mapping"] = Devise.mappings[:user]
-    @user = FactoryBot.create(:certificate_manager_user)
+    if (team.nil?) then team = FactoryBot.create(:team) end
```

Comment:
this could be also written as 
```suggestion
    team = FactoryBot.create(:team) if team.nil?
```